Userland/Utilities/icc.cpp

```diff
@@ -10,6 +10,16 @@
 #include <LibCore/MappedFile.h>
 #include <LibGfx/ICCProfile.h>
 
+template<class T>
+static void out_optional(char const* label, Optional<T> optional)
+{
+    out("{}: ", label);
+    if (optional.has_value())
+        outln("{}", *optional);
+    else
+        outln("(not set)");
+}
```

Comment:
:thonk: Maybe it'd be better to add a `Formatter<Optional>` that does something like this... though I guess the text you want for "empty" might be different for each use case.